benchmarking-tests/benchmark_utils.py

```diff
@@ -41,7 +43,7 @@ def create_write_transaction(tx_left, payload_filler):
 
 
 def run_add_backlog(args):
-    tx_left = args.num_transactions // mp.cpu_count()
+    tx_left = args.num_transactions / mp.cpu_count()
```

Comment:
As I understand it, in Python 3, and in Python 2.2+ with `from __future__ import division`, `/` means "[true division](http://www.informit.com/articles/article.aspx?p=1439189)", e.g.:
```python
In [1]: 1/2
Out[1]: 0.5

In [2]: 3/2
Out[2]: 1.5

In [3]: 12/10
Out[3]: 1.2

In [4]: 10/5
Out[4]: 2.0
```

In other words, even if the operands are integers, the result is always a float.

Most BigchainDB code is Python 3. (An exception is the AWS deployment scripts, which use Python 2 because Fabric doesn't work with Python 3 yet.) Therefore we don't need `from __future__ import division` for `/` to mean true division.

That said, I don't think we want true division here. We don't want `tx_left` to be a floating point number like 1.2, do we?

`//` will do "floor division" so the original version of this line would always result in an integer value for `tx_left`.